docs/README.md

```diff
@@ -124,17 +132,40 @@ productSearch(
     If you want faster searches and do not care about most up to date prices and promotions, use skip value.
     """
     simulationBehavior: SimulationBehavior = default
+    """
+    If true, all the products will be solved by VTEX API.
+    """
+    productOriginVtex: Boolean = false
```

Comment:
This is not clear

Reply:
I've updated the `search-protocol` and the comment in the `schema.graphql`. What do you think?

![image](https://user-images.githubusercontent.com/40380674/77179961-04744480-6aa8-11ea-8d38-ea1d0d05c5bd.png)

Reply:
Perfect! thanks!